### DISCUSSION TITLE
Webhook limit when creating more than 3 tags?

### DISCUSSION THREAD

**Discussion by greensheep** [community/community#24353] [Apps, API and Webhooks]
<p>I’ve noticed that if I create more than three tags in a single push, no webhooks are triggered.</p>
<p>For example, this works and three webhooks are delivered (ie, they show in the project’s webhook settings page and I receive them ok):</p>
<pre><code>git tag @scope/project-1@1.0.0-alpha.0 -m @scope/project-1@1.0.0-alpha.0
git tag @scope/project-2@1.0.0-alpha.0 -m @scope/project-2@1.0.0-alpha.0
git tag @scope/project-3@1.0.0-alpha.0 -m @scope/project-3@1.0.0-alpha.0
git push --tags
</code></pre>
<p>However this does not (tags are added to the repo but no webhooks are delivered - they’re not visible in the settings UI or received at my end):</p>
<pre><code>git tag @scope/project-1@1.0.0-alpha.0 -m @scope/project-1@1.0.0-alpha.0
git tag @scope/project-2@1.0.0-alpha.0 -m @scope/project-2@1.0.0-alpha.0
git tag @scope/project-3@1.0.0-alpha.0 -m @scope/project-3@1.0.0-alpha.0
git tag @scope/project-4@1.0.0-alpha.0 -m @scope/project-4@1.0.0-alpha.0
git push --tags
</code></pre>
<p>Is this the expected behaviour (eg, because of a rate limit)? If so, is there a way I can avoid this? (eg, batching tag pushes, upgrading my account, etc).</p>

**Comment by greensheep** [comment, marked as answer]
<p>As it turns out there <em>is</em> a limit of 3 on the number of tags that can be pushed. This is documented here:</p>
<p><a href="https://developer.github.com/v3/activity/events/types/#createevent" rel="nofollow noopener">https://developer.github.com/v3/activity/events/types/#createevent</a></p>
<p>Thanks to Github support for this info :+1:</p>

**Comment by thiagolinsx** [comment]
<p>How did you get around that limit? I’m using monorepo and eventually I commit with more than 3 releases … When that happens the actions don’t start.</p>

**Comment by fernandrone** [comment]
If anyone else hits this limit and ends up in this thread, we had this same issue and got around it using the solution proposed here https://github.com/lerna/lerna/issues/2218#issuecomment-771876933
